src/api/helpers/map-CIS-countries/index.ts

```diff
@@ -0,0 +1,32 @@
+import { EXTERNAL_API_DEFINITIONS } from '../../constants';
+import mapCisCountry from './map-CIS-country';
+import sortArrayAlphabetically from '../sort-array-alphabetically';
+import { CisCountry } from '../../types';
+
+const { CIS } = EXTERNAL_API_DEFINITIONS;
+
+/**
+ * filterCisCountries
+ * Filter out countries from CIS API that have an invalid name
+ * @param {Array} All CIS countries
```

Comment:
not a blocker, should we set this as `Array<CisCountry>`

Reply:
Great shout, i've added a return type as well.